### ISSUE TITLE
`rebar3 clean -a` fails because of a maps:from_list in provider_asn1_util

### ISSUE THREAD

**Issue by tsloughter** [knusbaum/provider_asn1#11]
Args are a proplist which can have bare atoms so `maps:from_list` will fail in cases like `rebar3 clean -a`:

```
Error: badarg
[{maps,from_list,[[all]],[]},
 {provider_asn1_util,resolve_args,2,
                     [{file,"/home/tristan/Devel/service_discovery/_build/default/plugins/provider_asn1/src/provider_asn1_util.erl"},
                      {line,53}]},
 {provider_asn1_clean,do,1,
                      [{file,"/home/tristan/Devel/service_discovery/_build/default/plugins/provider_asn1/src/provider_asn1_clean.erl"},
                       {line,39}]},
```

**Comment by knusbaum** [comment]
@tsloughter Can you confirm if #15 fixed this?

Sorry for the very long response time on this bug.

**Comment by knusbaum** [comment]
Just published `0.3.0` with the latest changes.

**Comment by sebastiw** [comment]
I'll close this as, 1. stale issue; no reply. 2. verified locally.

```
$ rebar3 compile
===> Verifying dependencies...
===> Fetching providers v1.8.1
===> Fetching getopt v1.0.1
===> Analyzing applications...
===> Compiling getopt
===> Compiling providers
===> Analyzing applications...
===> Compiling provider_asn1
$ rebar3 clean -a
===> Verifying dependencies...
===> Cleaning out getopt...
===> Cleaning out provider_asn1...
===> Cleaning out providers...
$
```
